doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-dmn-based-proc.adoc

```diff
@@ -1,7 +1,9 @@
 [id='test-scenarios-coverage-report-dmn-based-proc']
 = Generating coverage reports for DMN-based test scenarios
 
-Follow the procedure below to generate a coverage report for DMN-based test scenarios.
+In DMN-based test scenarios, the *Coverage Report* tab contains the detailed information about the number of available decisions, number of executed decisions, percentage of executed decisions, how many times each decision has been executed, and which decisions are executed for each defined test scenario in the table. The percentage of the executed decisions is also represented as a pie chart in the same *Coverage Report* tab.
```

Comment:
Suggestion - I think the sentence is too long. How about listing the information?

"In DMN-based test scenarios, the *Coverage Report* tab contains information about the following:
- Number of available decisions
- Number of executed decisions
- Percentage of executed decisions
- Percentage of executed decisions represented as a pie chart
- Number of times each decision has executed
- Which decisions are executed for each defined test scenario"